src/components/message/message.style.js

```diff
@@ -4,13 +4,20 @@ import { margin } from "styled-system";
 import BaseTheme from "../../style/themes/base";
```

Comment:
You need to remove this import and the `theme` default prop below

Reply:
When I'm removing theme from there all tests from message.spec.js from function testStyledSystemMargin are failing. I got info that we cannot delete that if styles are overwritten in another component which use theme and when we use margin/space/padding from styled-system since they are using theme.space

Reply:
Ah yeah good point, I didn't notice the styled-system margin in here before, that's fine then 👍🏻